content/blog/accessing-hpe-data-fabric-s3-storage-from-spring-boot-s3-micro-service-deployed-in-k3s-cluster.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Accessing HPE Data Fabric S3 Storage from Spring Boot S3 Micro Service
+  deployed in K3S cluster
+date: 2021-09-13T08:13:40.734Z
+featuredBlog: true
+priority: 0
+author: Kiran Kumar Mavatoor
+authorimage: /img/hpe-dev-grommet-gremlin-rockin.svg
+tags:
+  - K3S
+  - Spring Boot
+  - HPE Data Fabric S3 Storage
+---
+S3 is a storage service offered by HPE Data Fabric Platform that offers scalable, secure, and well performing object storage. In this article, we will discuss as how to access HPE Data Fabric Object Store (S3) using Spring Boot S3 Micro Service application deployed in [K3S cluster](https://k3s.io/) and perform basic S3 operations like upload, list, delete etc. Below diagram gives the architecture overview 
+
+![Timeline
+
+Description automatically generated](blob:https://hpe-dev-portal.netlify.app/48945ed2-01ab-482a-b7f7-fd955b0b51bd)
+
+Figure 1:  Architecture overview of Spring Boot S3 Micro Service on K3S with HPE Data Fabric S3 Storage as back end.
+
+The brief description of technology stack used is described below sections :
+
+**Data Fabric Object Storage**
+
+The Object Store with S3-Compatible API stores data generated through multiple data protocols, such as NFS, POSIX, S3, and HDFS. Data in the Object Store is accessible through S3 API requests. The Object Store manages all inbound S3 API requests to store data in or retrieve data from an HPE Ezmeral Data Fabric cluster. More details can be found at <https://docs.datafabric.hpe.com/62/MapRObjectStore/MapRObjectStorewithS3-compatibleAPI.html>
+
+**K3S**
+
+K3S is Lightweight Kubernetes. Easy to install, half the memory, all in a binary of less than 100 MB. Its Great for Edge and IOT use cases. More information on K3S can be found at <https://rancher.com/docs/k3s/latest/en/> . Follow the steps as mentioned in <https://rancher.com/docs/k3s/latest/en/quick-start/> for installation of K3S cluster. 
+
+**Spring Boot**
+
+Spring Boot is an open source Java-based framework used to create a micro Service. Many Real world applications are written in Spring Boot for faster development and better maintainability. More information can be found at** <https://spring.io/projects/spring-boot>**
+
+Note: We can move the Spring Boot application to Quarkus with no change in code.  Moving code to Quarkus will reduce the footprint of Spring Boot application. More information can be found at <https://quarkus.io/blog/quarkus-for-spring-developers/>
+
+In this blog, we will checkout Spring Boot application from Git , customise it and execute. 
+
+**Application Prerequisites**
+
+<!--\\\[if !supportLists]-->a.      <!--\\\[endif]-->K3S cluster must be accessible. Note down the control plan node details. This information is required to deploy the Spring Boot application
```

Comment:
Could you try using a markdown list in place of this, see if it looks better. I don;t understand how you got this syntax. Where is this markdown ported from?

Reply:
If you copied/paste from an existing document, make sure you paste as Plain text (CTRL+SHIFT+V).

Also, for consistency throughout your post, I would recommend to use numbered bullet list (1. 2. 3., etc) insted of a., b.,..

Put a dot "." at the end of each numbered list items in your lists.